lua/packer/display.lua

```diff
@@ -707,6 +721,14 @@ display.prompt_revert = function()
   if display.status.disp then display.status.disp:prompt_revert() end
 end
 
+display.retry = function()
+  if display.status.any_failed_install then
+      require("packer").install()
+  elseif #display.status.failed_update_list > 0 then
+      require("packer").update(unpack(display.status.failed_update_list))
```

Comment:
Is there a reason to build the failed updates list but not do the same for installs? Both `packer.install` and `packer.update` should take a list of plugins to operate upon. It's not a big deal for installs, but it does save some effort i.e. finding missing plugins.

Reply:
I was not sure what to do in case luarock packages failed and I cannot test it because I never used those. So, since ``packer.install`` already finds all missing installations, hopefully including luarock packages, I left the default one.